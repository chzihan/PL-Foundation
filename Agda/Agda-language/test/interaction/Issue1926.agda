module _ (A : Set) where

record R : Set where
  field f : A

test : R â†’ R
test r = {!r!}
