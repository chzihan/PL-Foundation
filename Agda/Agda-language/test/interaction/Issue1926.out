(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-status-action "")
(agda2-info-action "*All Goals*" "?0 : R " nil)
((last . 1) . (agda2-goals-action '(0)))
(agda2-status-action "")
(agda2-info-action "*Module contents*" "Modules Names f : A" nil)
((last . 1) . (agda2-goals-action '(0)))
